pallet/message-gadget/src/tests.rs

```diff
@@ -16,8 +16,9 @@
 // You should have received a copy of the GNU General Public License
 // along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
 
+use crate as darwinia_message_gadget;
```

Comment:
Format

Reply:
I am not fully understand the format problem here. Could you please review again and explain more?